base/poco/XML/CMakeLists.txt

```diff
@@ -44,6 +45,8 @@ target_compile_options (_poco_xml
         -Wno-unused-macros
         -Wno-unused-parameter
         -Wno-zero-as-null-pointer-constant
+        -Wno-covered-switch-default
+        -Wno-unreachable-code-break
```

Comment:
(funny that `-Wno-unreachable-code` (l. 44) doesn't imply `-Wno-unreachable-code-break`)